src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/Runners/LoggingRunnerAdapter.cs

```diff
@@ -187,7 +193,12 @@ private async Task ReadLinesAsync(StreamReader reader, List<string> lines, Actio
                 // stream if readAborded is not set. This is so we can ensure that the entire stream is read.
                 if (!readAborted && _finishReads)
                 {
+#if NET7_0_OR_GREATER
+                    string remainder = await reader.ReadToEndAsync(cancelToken).ConfigureAwait(false);
+#else
+                    // ReadLineAsync does not have cancellation in 6.0 or lower
```

Comment:
nit: this comment appears to be copied from above where we use `ReadLineAsync` instead of `ReadToEndAsync`